src/utils/__test__/mapsheet.test.ts

```diff
@@ -44,12 +44,18 @@ describe('MapSheets', () => {
     assert.deepEqual(MapSheet.offset('BW14'), { x: 1324000, y: 5226000 });
   });
 
-  for (const ms of MapSheetData) {
-    it('should calculate for ' + ms.code, () => {
+  it('should calculate for all sheets', () => {
+    for (const ms of MapSheetData) {
       assert.deepEqual(MapSheet.offset(ms.code), ms.origin);
       assert.equal(MapSheet.isKnown(ms.code), true);
-    });
-  }
+    }
+  });
+
+  it('should round trip all sheets', () => {
+    for (const ms of MapSheetData) {
+      assert.equal(MapSheet.sheetCode(ms.origin.x, ms.origin.y), ms.code);
```

Comment:
Do we also want a test for some codes outside of `MapSheetData`, like the eight quadrants around the map? Or if `MapSheetData` does not represent a square, just a couple of random points outside of it?

Reply:
In the next pull request I was going to add some logic for tiling outside the known range where I am adding more tests into this space.